.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "ethereum-tests"]
 	path = packages/ethereum-tests
 	url = https://github.com/ethereum/tests.git
-	branch = develop
+	branch = v11
```

Comment:
Will this branch regularly get updated or should we revert back to the `develop` branch? (If that has the latest tests we should do so)

Reply:
We should stay on `develop` methinks.  All the v11 tests have been merged in develop so as long as we pin the commit 95a8490f1f202dc9f58e910571d5d02082e0e641 that corresponds to the v11 release.

Reply:
Just checked, `develop` has new commits and `v11` has not, so we should stay on develop